lib/wallaby/dsl/finders.ex

```diff
@@ -13,6 +27,15 @@ defmodule Wallaby.DSL.Finders do
     do_find_elements(session, to_params(query))
   end
 
+  def any?(session, query) do
+    retry fn ->
+      case do_find_elements(session, to_params(query)) do
+        elements when length(elements) > 0 -> elements
```

Comment:
I think that this should return a boolean instead of the elements if its a `?` function.